src/nvim/os_unix.c

```diff
@@ -355,12 +355,12 @@ int mch_expand_wildcards(int num_pat, char_u **pat, int *num_file,
           }
           j++;
         } else if (!intick
-            && ((flags & EW_KEEPDOLLAR) == 0 || pat[i][j] != '$')
-            && vim_strchr(SHELL_SPECIAL, pat[i][j]) != NULL) {
-          // Put a backslash before a special character, but not
-          // when inside ``. And not for $var when EW_KEEPDOLLAR is
-          // set.
-          *p++ = '\\';
+          && ((flags & EW_KEEPDOLLAR) == 0 || pat[i][j] != '$')
```

Comment:
The linter is complaining about the following:

```
src/nvim/os_unix.c:358:  Inner expression should be aligned as opening brace + 1 (+ 2 in case of {)  [whitespace/alignment] [2]
src/nvim/os_unix.c:359:  Inner expression should be aligned as opening brace + 1 (+ 2 in case of {)  [whitespace/alignment] [2]

```

Any idea how to make it happy again? @justinmk maybe?